downloadaudio/download.py

```diff
@@ -50,6 +51,9 @@
 from .review_gui import review_entries
 from .update_gui import update_data
 
+DOWNLOAD_NOTE_SHORTCUT  = "q"
+DOWNLOAD_SIDE_SHORTCUT  = "Shift+W"
+DOWNLOAD_MANUAL_SHORTCUT  = "Ctrl+w"
```

Comment:
For me, `Ctrl+w` means “close document”, in Anki, return to the deck list. See my [shortcut add-on](https://github.com/ospalh/anki-addons/blob/develop/more_shortcuts.py#L31).